guides/common/modules/ref_best-practices-for-sync-plans.adoc

```diff
@@ -0,0 +1,24 @@
+[id="best-practices-for-sync-plans_{context}"]
+= Best practices for sync plans
+
+* Add sync plans to products and regularly synchronize content to avoid overly large changes in repositories.
+Synchronize content rather more often than less often.
+For example, setup a sync plan to synchronize content every day rather than only once a month.
+* Automate the creation and update of sync plans by using a Hammer script or an {ansible-docs-url}[Ansible playbook].
+* Distribute synchronization tasks over several hours to reduce the task load by creating multiple sync plans with *Custom Cron*.
+* Use *Custom Cron* to specify recurring synchronization using cron expressions.
```

Comment:
I'm not sure you need this line. The name of the tool pretty much says it all. :-)

Reply:
Removed.